src/openforms/config/models/config.py

```diff
@@ -576,6 +576,14 @@ class GlobalConfiguration(SingletonModel):
         max_length=100,
         blank=True,
     )
+    communication_preferences_portal_url = models.URLField(
+        _("communication preferences portal url"),
+        blank=True,
+        help_text=_(
+            "Url to the online portal where users can update their communication preferences. "
+            "This is used in conjunction with the communication preferences (customer interactions API) prefill in the profile component."
```

Comment:
```suggestion
            "This is used by the customer profile component."
```
- simpler language for accessibility 
- it's not used in/during prefill, only exposed in the form detail endpoint for the customer profile component

Reply:
I meant it more as "this is only used when there are prefilled/fetched digital addresses", which happens using the communication preferences prefill.

But it does complicate things.. Changing the message